src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -139,8 +147,9 @@ impl<F: TableBuilderFactory> CapacitySplitTableBuilder<F> {
                 table_ids,
             };
             let policy = self.policy;
-            let tracker = self.tracker.take();
+            let mut tracker = self.tracker.take().unwrap();
             let upload_join_handle = tokio::spawn(async move {
+                tracker.try_increase_memory(data.capacity() as u64 + meta.encoded_size() as u64);
```

Comment:
We may have a warning log when `try_increase_memory` fails so that in case that the real memory usage exceeds the memory limit, we may check the log to see whether it's because `try_increase_memory` fails.

Reply:
No. I do not think it need a warn log because it is difficult to limit strict the real memory....
debug log is enough.